src/Features/Auth/ForgetPassword/ForgetPasswordContainer.js

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { Typography, Grid, Box } from '@material-ui/core';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router';
 import { Link } from 'react-router-dom';
 
 import Button from '../../../components/Button/Button';
 import Card from '../../../components/Card/Card';
 import TextField from '../../../components/TextField/TextField';
+import { forgotPassword } from '../actions';
+
 function ForgetPasswordContainer() {
+  const [login, setLogin] = useState({
+    email: '',
+    password: '',
+  });
+
+  const history = useHistory();
+
+  const dispatch = useDispatch();
+
+  const { message, status } = useSelector((state) => {
+    const { message, status } = state.responseMessage;
+    return { message, status };
+  });
+
+  const sendEmail = () => {
+    dispatch(forgotPassword(login));
+  };
+  useEffect(() => {
+    if (status === 200) {
+      history.push('/login');
+    }
+  });
+  function loginForm(e) {
```

Comment:
This function is supposed to render loginForm according to the name but it takes input from field and set in State. You should rename this function.